### PR TITLE
added toast and rating component

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs
## Screenshot:
toast-and-rating

## Deploy Preview Link:
https://deploy-preview-11--comp-ui.netlify.app/

### CODE REVIEW

File: components/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.rating{
+    margin: 0.5em;
+    font-size:1.5em;
+    direction: rtl;
+    text-align: left;
+}
+.rating > span {
```
Comment (mohitdhatrak) [inline components/rating/rating.css:7]: add 1 line space after each css block for better readability

File: components/rating/rating.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        </p>
+        <h2 class="sub-comp-header">Simple Rating</h2>
+
+        <div class="comp-disp-container">
+            <div class="rating">
+                <span>☆</span>
+                <span>☆</span>
+                <span>☆</span>
+                <span>☆</span>
+                <span>☆</span>
```
Comment (mohitdhatrak) [inline components/rating/rating.html:108]: why not use icons? Why put the stars as text. Just asking, no issues if you want this. Maybe icons would be better i think

File: components/toast/toast.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                    <h2 class="sub-comp-header">Simple Toast</h2>
+                    <h2 class="sub-comp-header">&times</h2>    
+                </div>
+                <div class="toast-text">
+                    <p>    
+                    The toast component is like an alert box that is only shown 
+                    for a couple of seconds when something happens (i.e. when the 
+                    user clicks on a button, submits a form, etc.).
+                    </p>
+                </div>
```
Comment (mohitdhatrak) [inline components/toast/toast.html]: this div is not necessary i feel, as only p tag is inside it. Directly give class to p itself.

**Review by mohitdhatrak — COMMENTED** [review]
Looks good overall good job. Few suggestions here and there from me.

File: components/rating/rating.html (hunk 2 above)
Comment (tushargoyal812) [reply components/rating/rating.html:108]: Suggestion:use font awesome icons its better.

File: components/toast/toast.css [hunk 4]
```diff
@@ -0,0 +1,14 @@
+:root{
+    --secondary-color: #e1f1ef;
+}
+.toast {
+    background-color: var(--secondary-color);
+    border-left: 10px solid var(--primary-color);
+    padding: 0.1em 1em;
+    border-radius: 0.6em;
```
Comment (tushargoyal812) [inline components/toast/toast.css:8]: give border radius in px as vikram suggested.

**Review by tushargoyal812 — COMMENTED** [review]
just take care of some suggestions otherwise looks good to merge

File: components/rating/rating.css (hunk 1 above)
Comment (bishnoimukesh) [reply components/rating/rating.css:7]: Ok, will take care of it.

File: components/rating/rating.html (hunk 2 above)
Comment (bishnoimukesh) [reply components/rating/rating.html:108]: I had used icons enough to try something with text. Further, I will use the icons. 
Thanks for the suggestions.

File: components/toast/toast.html (hunk 3 above)
Comment (bishnoimukesh) [reply components/toast/toast.html]: Done

**Pull request merged**